### PR TITLE
first update

### PR DESCRIPTION
**Pull request by minwoos2**
Created heatequation, plotted, and created an animation

### CODE REVIEW

**Comment by minwoos2** [comment]
I have reviewed and hereby sign the CLA

File: heatequation.jl [hunk 1]
```diff
@@ -0,0 +1,71 @@
+using OrdinaryDiffEq, ModelingToolkit, DiffEqOperators, DomainSets
+# Method of Manufactured Solutions: exact solution
+u_exact = (x,t) -> exp.(-t) * cos.(x)
```
Comment (ctessum) [inline heatequation.jl:3]: I think we're not using this any more. If so, it can be deleted.

File: heatequation.jl [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Dyy = Differential(y)^2
+Dt = Differential(t)
+t_min= 0.
+t_max = 2.0
+x_min = 0.
+x_max = 2.
+y_min = 0.
+y_max = 2.
+
+#xmax = 50.0
```
Comment (ctessum) [inline heatequation.jl:17]: You can delete these variables we're not using any more

File: heatequation.jl [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+#ymax = 30.0
+#ncells = 50.0
+#dx = x_max / ncells
+
+
+lecturer(x, y) = x > x_max / 2 - dx && x < x_max / 2 + dx && y > y_max / 10 - dx && y < y_max / 10 + dx
+emission(x, y, emisrate) = lecturer(x, y) ? emisrate : 0
+@register emission(x, y, emisrate)
+emisrate = 10;
```
Comment (ctessum) [inline heatequation.jl:26]: Do you need to define emisrate above where it is used? It seems like how it is now might cause an error if you run the file from beginning to end.

File: heatequation.jl [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+
+
+# 3D PDE
+eq  = Dt(u(t,x,y)) ~ Dxx(u(t,x,y)) + Dyy(u(t,x,y)) + emission(x, y, emisrate)
+#emission(x, y, emisrate) = emisrate
+
+analytic_sol_func(t,x,y) = exp(x+y)*cos(x+y+4t)
```
Comment (ctessum) [inline heatequation.jl:34]: You can delete this and related things below if they're not being used any more

File: heatequation.jl [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+           y ∈ IntervalDomain(y_min,y_max)]
+pdesys = PDESystem([eq],bcs,domains,[t,x,y],[u(t,x,y)])
+
+# Method of lines discretization
+dx = 0.1; dy = 0.1
+discretization = MOLFiniteDifference([x=>dx,y=>dy],t)
+prob = ModelingToolkit.discretize(pdesys,discretization)
+sol = solve(prob,Tsit5())
+
+# Ploting
```
Comment (ctessum) [inline heatequation.jl:56]: Ploting -> Plotting

**Review by ctessum — CHANGES_REQUESTED** [review]
Looking good!

File: heatequation.jl [hunk 6]
```diff
@@ -0,0 +1,71 @@
+using OrdinaryDiffEq, ModelingToolkit, DiffEqOperators, DomainSets
```
Comment (ctessum) [inline heatequation.jl:1]: Since you've added the file with the different name, you can also remove this file.

**Pull request merged**